File: backend/src/main/java/com/ll/nbe342team8/domain/book/review/dto/ReviewRequestDto.java
package com.ll.nbe342team8.domain.book.review.dto;

import com.ll.nbe342team8.domain.book.review.entity.Review;

import java.time.LocalDateTime;

public record ReviewRequestDto(
        String content,
        Double rating
) {

    public static ReviewRequestDto from(Review review){
        return new ReviewRequestDto(
                review.getContent(),
                review.getRating()
        );
    }
}

File: backend/src/main/java/com/ll/nbe342team8/domain/book/review/dto/ReviewRequestDto.kt
package com.ll.nbe342team8.domain.book.review.dto

import com.ll.nbe342team8.domain.book.review.entity.Review

@JvmRecord
data class ReviewRequestDto(
    val content: String,
    val rating: Double
) {
    companion object {
        fun from(review: Review): ReviewRequestDto {
            return ReviewRequestDto(
                review.content,
                review.rating
            )
        }
    }
}

